src/main/java/uk/ac/ic/wlgitbridge/git/handler/WLRepositoryResolver.java

```diff
@@ -82,6 +83,23 @@ public Repository open(
              ServiceNotAuthorizedException,
              ServiceMayNotContinueException {
         Log.info("[{}] Request to open git repo", name);
+        // Reject v1 ids, the request will be rejected by v1 anyway
+        if (name.matches("^[0-9]+[bcdfghjkmnpqrstvwxyz]+$") && !name.matches("^[0-9a-f]{24}$")) {
```

Comment:
In the distant past, we also allowed `l` (lower case L), so we should add that to the regex for the letter part. We could also restrict the length of the letter part to be between 6 and 12 characters inclusive, but that is less important.

Reply:
:+1: